RecyclerViewClickHandler-Starter/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
         room_version = '2.2.0'
         coroutine_version = '1.3.7'
         gradleVersion = '4.0.1'
-        navigationVersion = '1.0.0'
+        navigationVersion = '1.0.0-alpha07'
```

Comment:
Is it 2.3.0 now? https://developer.android.com/jetpack/androidx/releases/navigation

Reply:
This need some code changes and may be codelab changes as well. So it is best to do it in a separate CL